ocs_ci/utility/utils.py

```diff
@@ -1303,3 +1304,39 @@ def wait_for_co(operator):
     from ocs_ci.ocs.ocp import OCP
     ocp = OCP(kind='ClusterOperator')
     ocp.get(operator)
+
+
+def censore_values(data_to_censore, keys):
```

Comment:
*censor

Reply:
Thanks, I didn't check the dictionary and just blindly thought from word censored that it's censore but obviously it's not ;) Will correct on all places.